.github/workflows/sync_upstream.yaml

```diff
@@ -35,4 +35,13 @@ jobs:
           git merge --allow-unrelated-histories --no-edit upstream/master
           git push origin master 
 
+      - name: On Fail, Send Message About Merge to Slack
+        if: ${{ failure() }}
+        uses: archive/github-actions-slack@v2.8.0
+        id: notify-slack
+        with:
+          slack-bot-user-oauth-access-token: ${{ secrets.SLACK_BOT_TOKEN }}
+          slack-channel: ${{ secrets.SLACK_BOT_INTEGRATION_INNOVATORS_CHANNEL }} #USE CHANNEL ID, NOT CHANNEL NAME, SINCE ID IS USED IN NEW SLACK API's
```

Comment:
should this not go in the generic alerts channel?

Reply:
ticket says support should be notified maybe talk to Andy about where to put it?